### ISSUE TITLE
1

### ISSUE THREAD

**Issue by CR3Swapper** [wangrongding/github-old-feed#1]
![image](https://github.com/wangrongding/github-old-feed/assets/137743565/c90b36bb-45a3-4e45-8f6e-3fec265e6333)

**Comment by blyxyas** [comment]
Yep, I'm sure nobody in the GH staff expected people to have strong opinions about this topic. Really it's just a CSS change, as you can use filters to customize your experience.

**Comment by Freddd13** [comment]
There's no better word than shit to evaluate the new feed.
